Source/WebCore/platform/sql/SQLiteDatabase.h

```diff
@@ -59,7 +60,11 @@ class SQLiteDatabase : public CanMakeWeakPtr<SQLiteDatabase> {
     static constexpr ASCIILiteral inMemoryPath() { return ":memory:"_s; }
 
     enum class OpenMode { ReadOnly, ReadWrite, ReadWriteCreate };
-    WEBCORE_EXPORT bool open(const String& filename, OpenMode = OpenMode::ReadWriteCreate);
+    enum class OpenOptions {
```

Comment:
Maybe:
`enum class OpenOptions : uint8_t {`

(And maybe this same change to OpenMode, since it only has three values).

But not needed for this change.